.github/workflows/build.yml

```diff
@@ -31,6 +31,13 @@ jobs:
         uses: actions/checkout@v2
       - name: yaml-lint
         uses: ibiqlik/action-yamllint@v3
+      - name: Check goreleaser config
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: v0.179.0
+          args: check
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
adding goreleaser file check earlier in the build, this adds ~1s to the build time